src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/DedupJobSchedulerDAO.java

```diff
@@ -0,0 +1,141 @@
+package com.lithium.dbi.rdbi.recipes.scheduler;
+
+import com.lithium.dbi.rdbi.BindArg;
+import com.lithium.dbi.rdbi.BindKey;
+import com.lithium.dbi.rdbi.Mapper;
+import com.lithium.dbi.rdbi.Query;
+
+import java.util.List;
+
+/**
+ * Main difference between this DAO and the ExclusiveJobSchedulerDAO is that this DAO
+ * allows the same job to be in the running queue and ready queue at the same time.
+ *
+ * Also while reserving jobs, a job is only reserved if the job has no instances in the
+ * running queue.
+ */
+public interface DedupJobSchedulerDAO {
+    /**
+     * When scheduling a job make sure its not already in the ready queue. A job
+     * can be scheduled regardless of whether its in the running queue.
+     */
+    @Query(
+        "local readyJobScore = redis.call('ZSCORE', $readyQueue$, $jobStr$)\n" +
+        "local isPaused = redis.call('GET', $pausedTube$) \n" +
+        "if not readyJobScore and not isPaused then\n" +
```

Comment:
no numeric comparison here, but in deleteJob you chose to add that comparison. why ?